src/plots/cartesian/layout_attributes.js

```diff
@@ -492,6 +492,34 @@ module.exports = {
             '*%H~%M~%S.%2f* would display *09~15~23.46*'
         ].join(' ')
     },
+    tickformatstops: {
+        _isLinkedToArray: 'tickformatstop',
+
+        dtickrange: {
+            valType: 'info_array',
+            role: 'info',
+            items: [
+                {valType: 'any', editType: 'ticks'},
+                {valType: 'any', editType: 'ticks'}
+            ],
+            editType: 'ticks',
+            description: [
+                'range [*min*, *max*], where *min*, *max* - dtick values',
+                'which describe some zoom level, it is possible to omit *min*',
+                'or *max* value by passing *null*'
+            ].join(' ')
+        },
+        value: {
+            valType: 'string',
+            dflt: '',
```

Comment:
Is `axis.tickformat: ''` a valid format? More precisely, could

```js
tickformatstop: [{
  dtickrange: [/* */],
  value: ''
}]
```

be used to hide tick labels for some dtick range?

Reply:
no, `tickformat: ''` gives our default formatting. I suppose we could imagine making our own "hide" tickformat (for date axes you can already hack this with `tickformat: ' '` but that doesn't work for numbers) but it seems a bit of a kludgy way to handle a very esoteric edge case.